games/DuckHunt-JS/dist/index.html

```diff
@@ -22,6 +22,16 @@
       background-color: #000000;
     }
   </style>
+  
+  <!-- Google tag (gtag.js) -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-L26FXBG7SW"></script>
```

Comment:
Gtag code not changed here?

Reply:
Forgot to save that code. Committed that code as well